apps/src/templates/teacherDashboard/shapes.jsx

```diff
@@ -5,6 +5,7 @@ import {makeEnum} from '@cdo/apps/utils';
 export const sectionShape = PropTypes.shape({
   id: PropTypes.number.isRequired,
   name: PropTypes.string.isRequired,
+  createdAt: PropTypes.string.isRequired,
```

Comment:
I suspect this being required will break tests or Stories where that field isn't provided, but we can add createdAt in those places if needed.